packages/cursorless-engine/src/snippets/updateSnippetBody.ts

```diff
@@ -0,0 +1,41 @@
+import { getLeadingWhitespace } from "../util/regex";
+
+export function updateSnippetBody(body: string): string {
```

Comment:
I think I might call this `normalizeSnippetBodyTabs` to make it super clear what's happening here. And a quick jsdoc would be nice